run.sh

```diff
@@ -346,6 +349,23 @@ check_plugins() {
   done <<< "$(WP plugin list --field=name)"
 }
 
+check_packages() {
+  if [[ ! "${REQUIRE-}" ]]; then
+    h3 "No package dependencies listed"
+    STATUS SKIP
+    return
+  fi
+
+  h3 "Installing composer packages"
+  touch /app/composer.json
```

Comment:
I agree we have too many ENV variables and that this one might confuse some people. My tendency would also have been to mount the file at a specified location (I wouldn't do a lookup for the file).

That said, I also see the argument of consistency and having everything configured in one place.

What I really hate though is generating the `compose.json` file from `run.sh`. Isn't there a `install` command for composer like for npm that would do that for us??